syntax = "proto3";
package cache;

// Service
service ProcessData {
  rpc processAOS(processAOSRequest) returns (processAOSResponse) {}
}

// Request Name
message processAOSRequest {
  bytes frame=1;
  bytes data=2;
}

// Response Array
message processAOSResponse{
  bytes frame=1;
  bytes data=2;
}